### ISSUE TITLE
Last line not true when FetchError

### ISSUE THREAD

**Issue by neminovno** [raineorshine/npm-check-updates#1127]
- [x] I have searched for [similar issues](https://github.com/raineorshine/npm-check-updates/issues)
- [x] I am using the latest version of `npm-check-updates`
- [x] I am using `node >= 14`

$ node -v
v16.15.0

$ npm -v
8.5.5

$ ncu --version
13.0.4

---

## Steps to Reproduce
1. Disconnect from internet
2. Run ncu

```
...
FetchError: Request ts-node info failed[3 of 3]:
request to https://registry.npmjs.org/ts-node failed, reason: getaddrinfo ENOTFOUND registry.npmjs.org.

FetchError: Request typescript info failed[3 of 3]:
request to https://registry.npmjs.org/typescript failed, reason: getaddrinfo ENOTFOUND registry.npmjs.org.

FetchError: Request webpack-bundle-analyzer info failed[3 of 3]:
request to https://registry.npmjs.org/webpack-bundle-analyzer failed, reason: getaddrinfo ENOTFOUND registry.npmjs.org.
[====================] 102/102 100%

All dependencies match the latest package versions :)
```

## Current Behavior
Last line is not true.

## Expected Behavior
To not show that line, or event better write what is wrong.

**Comment by raineorshine** [comment]
Thanks for reporting! I agree. I'll tidy that up.

**Comment by raineorshine** [comment]
Looks better now. Published in `v13.1.0`.
